code/modules/mob/living/basic/space_fauna/bear/bear_ai_behavior.dm

```diff
@@ -0,0 +1,66 @@
+/datum/ai_behavior/basic_melee_attack/bear
+	action_cooldown = 2 SECONDS
+
+/datum/ai_behavior/find_hunt_target/find_hive
+
+/datum/ai_behavior/find_hunt_target/find_hive/valid_dinner(mob/living/source, obj/structure/beebox/hive, radius)
+	if(!length(hive.honeycombs))
+		return FALSE
+	return can_see(source, hive, radius)
+
+/datum/ai_behavior/hunt_target/find_hive
+	always_reset_target = TRUE
+
+/datum/ai_behavior/hunt_target/find_hive/target_caught(mob/living/hunter, obj/structure/beebox/hive_target)
+	var/datum/callback/callback = CALLBACK(hunter, TYPE_PROC_REF(/mob/living/basic/bear, extract_combs), hive_target)
+	callback.Invoke()
+
+/datum/ai_behavior/find_and_set/valid_tree
+
+/datum/ai_behavior/find_and_set/valid_tree/search_tactic(datum/ai_controller/controller, locate_path, search_range)
+	var/list/valid_trees = list()
+	for (var/obj/structure/flora/tree/tree_target in oview(search_range, controller.pawn))
+		if(istype(tree_target, /obj/structure/flora/tree/dead)) //no died trees
+			continue
+		valid_trees += tree_target
+
+	if(valid_trees.len)
+		return pick(valid_trees)
+
+/datum/ai_behavior/climb_tree
+	behavior_flags = AI_BEHAVIOR_REQUIRE_MOVEMENT | AI_BEHAVIOR_REQUIRE_REACH
```

Comment:
this seems li ke something any basic mob can do (e.g. sloths are known for climbing trees), i think this and the tree finding behavior can be split out of this file and placed in a more generic file in the ai_behaviors folder without incident.